README.md

```diff
@@ -124,6 +124,34 @@ The possible transfers, and their codes, should be defined in the configuration.
 
 See **DoubleEntry::Transfer** for more info.
 
+### Metadata
+
+You may associate arbitrary metadata with transfers, for example:
+
+```ruby
+DoubleEntry.transfer(
+  Money.new(20_00),
+  :from => one_account,
+  :to   => another_account,
+  :code => :a_business_code_for_this_type_of_transfer,
+  :metadata => {:key1 => 'value 1', :key2 => 'value 2'},
+)
+```
+
+Metadata may be used for filtering lines for aggregation, for example:
```

Comment:
The reporting module is currently marked `@api_private` perhaps this shouldn't be described in the readme yet.